dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -347,6 +348,8 @@ public void visualizeSpecialBlock(SpecialBlock sbb) {
 
     @Override
     public void visualizeBlockTransferInput(Block bb, Analysis<A, S, T> analysis) {
+        assert analysis != null : "analysis should be non-null when visualize transfer input of a block";
```

Comment:
How about
`analysis should be non-null when visualizing the transfer input of a block`